docs/en/observability/whats-new.asciidoc

```diff
@@ -11,4 +13,278 @@ https://www.elastic.co/blog/elastic-observability-7-6-0-released[7.6] |
+
+Also, you can drag nodes into a structure that is easier for you to understand. To minimize the noise on the maps,
+and to make them easier to navigate, multiple external service calls are automatically collapsed into groups to
+display the relationships between sets of services and external connections.
+
+[role="screenshot"]
+image::images/service-map.png[APM service maps]
+
+[discrete]
+== Service inventory view with sparkline charts
+
+In {minor-version}, we have added health indicators to the service inventory view. Powered by the APM {ml} integration,
+these health indicators help spot services that need attention quickly. In addition to this, we have added a sparkline
+chart to the inventory view to help you understand the historical performance of your service.
+
+[role="screenshot"]
+image::images/service-inventory.png[Service inventory view]
+
+[discrete]
+== Anomaly detection alerts
+
+In {minor-version}, we have made the APM {ml} integration more actionable by adding the capability to create alerts
+on this data. While static thresholds are great for creating basic alerts, they can potentially cause a lot of noise
+when the underlying data is expected to be bursty. The new alert based on anomaly detection enables you to create alerts
+that take bursty or seasonal traffic patterns into account and notify a user only when true anomalies are detected.
+
+[role="screenshot"]
+image::images/anomaly-detect-alert.png[Anomaly detection alerts]
+
+[discrete]
+== APM PHP agent (beta)
+
+We are excited to introduce the PHP agent as a beta to Elastic APM. With this new agent, you get instant visibility
+into the performance of your PHP applications. The addition of the PHP agent now extends our support of all the major
+application development languages, which includes Java, .NET, GO, Python, Ruby, and Javascript.
+
+[discrete]
+== APM Java agent: Micrometer integration
+
+With the Java agent release 1.18.0, we have added support for automatic metrics collection from Micrometer. The
+metrics include cache statistics, thread pools, page hits, and also your custom business-related metrics, such as
+revenue which you can view in {kib}. For information on how to set up and start collecting these
+metrics, see our {apm-java-ref}/metrics.html#metrics-micrometer[Micrometer documentation].
+
+[discrete]
+== OpenTelemetry metrics intake
+
+We extended the https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/master/exporter/elasticexporter[Elastic exporter]
+for OpenTelemetry Collector to collect metrics and export them to the APM server. The Elastic OpenTelemetry exporter
+now supports both traces and metrics collection. OpenTelemetry metrics help to monitor business KPIs and technical
+components that are not auto instrumented by the OpenTelemetry SDK.
+
+Capturing insightful business metrics on applications with OpenTelemetry looks like this:
+
+[source,opentelemetry]
+----
+// initialise metric
```

Comment:
Sorry I wrote "initialize" in British English with an "s" instead of writing it in American English with a "z"

Reply:
Thanks @cyrille-leclerc! I updated the example.